test/mailers/bounty_mailer_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+class BountyMailerTest < ActionMailer::TestCase
+  def setup
+    # setup @user and @bounties here
+  end
+
+  test "bounty email" do
+    mail = BountyMailer.with(user: @user, bounties: @bounties).open_bounty
+
+    assert_equal ________, mail.subject
+    assert_equal _________, mail.to
+    assert_equal __________], mail.from
+    assert_match bounties.first.title, mail.body.encoded
+  end
+end
+
```

Comment:
It would be nice to have some basic test coverage for this.  I added a couple of examples to get you started for what I normally add when writing mailer tests.  You'll need to replace the `___________` lines with a string or value to make the test pass, something like:

`assert_equal 'Here are your bounties', mail.subject`

The setup section is also blank.  You can either create a User and Bounties manually or use fixtures (`test/fixtures.user.yml`)